python/cuml/internals/global_settings.py

```diff
@@ -23,8 +24,17 @@ class _GlobalSettingsData(threading.local):  # pylint: disable=R0903
 
     def __init__(self):
         super().__init__()
+        built_with_cuda = True
```

Comment:
Can we move this up to a module-level constant to make it easy to access elsewhere?